src/DataStructures/StripeIterator.hpp

```diff
@@ -8,9 +8,6 @@
 
 template <size_t>
 class Index;
-namespace PUP {
-class er;
-}  // namespace PUP
 
```

Comment:
Further down the class has
```
/// Needed for Charm++ serialization
StripeIterator() = delete;
```
which made no sense even before this change, but at least the comment should be removed.